corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -553,6 +555,11 @@ def add_no_items_text_to_detail(detail, app, detail_type, module):
         if detail_type.endswith('short') and app.supports_empty_case_list_text:
             detail.no_items_text = Text(locale_id=id_strings.no_items_text_detail(module))
 
+    @staticmethod
+    def add_select_text_to_detail(detail, app, detail_type, module):
+        if detail_type.endswith('short'):
```

Comment:
I thought i already left this review but i don't see it so apologies if it shows up twice.

We need to add a check of the CommCare build version (similar to what's done in `supports_empty_case_list_text`) since this will be added to all suite files. Otherwise, it'll break mobile builds sine it won't know how to parse that XML.